PrintMessage.cpp

```diff
@@ -0,0 +1,15 @@
+#include "./PrintMessage.h"
+#include <thread>
+#include <chrono>
+#include <iostream>
+using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
+
+void displayMessageandStars(const std::string& message) {
```

Comment:
Good approach to have this function in its own file. Then the display mechanism can evolve without disturbing the other logic.
One suggestion: The purpose of this function is to show the message and grab attention. By naming it `...Stars`, it "leaks" the knowledge of the display to the caller. So if star becomes hash, we need to change the function name and the calling cpp file as well.